src/components/EditSections/EditExtendedInfo/CreateResetPasswordControl/CreateResetPasswordControl.test.js

```diff
@@ -0,0 +1,54 @@
+
+import { screen, waitFor, cleanup } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import '__mock__/stripesCore.mock';
+
+import renderWithRouter from 'helpers/renderWithRouter';
+import CreateResetPasswordControl from './CreateResetPasswordControl';
+
+jest.unmock('@folio/stripes/components');
+jest.unmock('@folio/stripes/smart-components');
+
+const renderCreateResetPasswordControl = (props) => renderWithRouter(<CreateResetPasswordControl {...props} />);
+
+const propData = (postMock) => {
+  return {
+    email: 'testemail@email.com',
+    name: 'sample',
+    userId: 'testUserId123',
+    mutator: {
+      resetPassword: {
+        POST: postMock,
+      }
+    },
+  };
+};
+
+describe('CreateResetPasswordControl component', () => {
+  afterEach(cleanup);
+  it('If CreateResetPasswordControl Renders', () => {
+    const mockFunc = jest.fn(() => new Promise((resolve, _) => {
+      resolve({ ok: true, link: 'bl-users/password-reset/link' });
+    }));
+    renderCreateResetPasswordControl(propData(mockFunc));
+    expect(screen.getByText('ui-users.extended.sendResetPassword')).toBeInTheDocument();
+  });
+  it('If reset password renders on click', async () => {
+    const mockFunc = jest.fn(() => new Promise((resolve, _) => {
+      resolve({ ok: true, link: 'bl-users/password-reset/link' });
+    }));
+    await waitFor(() => renderCreateResetPasswordControl(propData(mockFunc)));
+    await userEvent.click(screen.getByText('ui-users.extended.sendResetPassword'));
+    expect(screen.getByText('testemail@email.com')).toBeInTheDocument();
```

Comment:
Are you confident this assertion is only valid after the click? Given the user-edit form is populated, I would expect the email address to be in the document both before and after clicking the button. Is there a different/better value we can assert on, i.e. one that is dependent on the click?

Reply:
Changed the assertion to copylink which would be rendered only on the click